styles/colors.js

```diff
@@ -0,0 +1,7 @@
+const colors = {
+  white: '#FFFFFF',
```

Comment:
What I meant with those colours was rather to make it 
```suggestion
  textColor: '#FFFFFF',
  backgroundColor: '#0B0A0A',
```